specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -5114,459 +5125,2029 @@
+          },
+          {
+            "name": "integrationServiceEnvironment",
+            "description": "The integration service environment.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationServiceEnvironment"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IntegrationServiceEnvironment"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/IntegrationServiceEnvironment"
+            }
+          },
+          "default": {
+            "description": "Logic error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
         }
       },
-      "x-ms-azure-resource": true,
-      "description": "The base resource type."
-    },
-    "SubResource": {
-      "properties": {
-        "id": {
+	    "patch": {
+        "tags": [
+          "IntegrationServiceEnvironments"
+        ],
+        "operationId": "IntegrationServiceEnvironments_Update",
+        "description": "Updates an integration service environment.",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Patch an integration service environment": {
+            "$ref": "./examples/IntegrationServiceEnvironments_Patch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationServiceEnvironmentName",
+            "description": "The integration service environment name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "integrationServiceEnvironment",
+            "description": "The integration service environment.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationServiceEnvironment"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IntegrationServiceEnvironment"
+            }
+          },
+          "default": {
+            "description": "Logic error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+	    "delete": {
```

Comment:
Does this delete all other resources which reference the ISE?

Reply:
Yes all the resources created in the ISE are deleted.

Reply:
What if the user does not have permissions to delete those resources? Is there a way from ISE to know all the resources which are created in ISE? I dont see it in the object model.

Reply:
It is as simple as when the container is getting deleted, all the artifacts within that container gets deleted. Now the user who is deleting this container knows that all LA etc created in this container will get deleted. Its the same principle followed everywhere else also.